gef-ui/templates/partials/upload-supporting-documents.njk

```diff
@@ -108,5 +108,21 @@
 {% endblock %}
 
 {% block scripts %}
-  <script src="/assets/js/fileUpload.js"></script>
+  <script>
+    if (typeof MOJFrontend.MultiFileUpload !== 'undefined') {
+      const onFileDelete = (mojFileUpload, response) => {
```

Comment:
This was the only way I could find to show an error message when the delete request fails. There didn't seem to be a nicer hook in.

Reply:
I've changed it so that when a delete request succeeds, all of the failed delete messages disappear.

I've had a play around with that and I think it all feels intuitive.

Reply:
This <script> needs to be moved to an external script file, CSP violation will be flagged on console.